src/models/admin.js

```diff
@@ -29,16 +29,16 @@ const userSchema = new mongoose.Schema({
     required: [true, 'Please enter a phone number'],
     min: 10,
   },
-  isEmailVerified:{
-    type: Boolean,
-    default: false
-  },
+  // isEmailVerified:{
+  //   type: Boolean,
+  //   default: false
+  // },
```

Comment:
Hello!!

What are you doing? Are you aware of the implication of removing **`isEmailVerified`** from the code?
If you are to remove Email Verification from Admin then all you need do is a single line comment on `app.js`  file.

Reply:
@HackinwaleDev I commented out the appropriate line in `app.js`. If commenting out lines `32-35` causes any breaking changes, please let me know so i undo it.